.travis.yml

```diff
@@ -25,5 +25,6 @@ before_script:
 - docker exec -it db timeout 60 bash -c 'while ! mysqladmin ping --silent; do sleep 3; done'
 script:
 - flake8 --exclude venv .
-- timeout 60 bash -c 'while ! tests/is_table_empty.py; do sleep 3; done'
+- bash tests/wait_mysql_up.sh
```

Comment:
bash is redundant if the script has execute permissions

Reply:
I had windows issues with the permission, this is why I used bash (and left it there).
solved it [this way](https://medium.com/@akash1233/change-file-permissions-when-working-with-git-repos-on-windows-ea22e34d5cee)